app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+
+<li>transcribe words and short sentences that they hear with increasing accuracy</li>
+
+<li>have conversations and make use of important social conventions such as formal modes of address</li>
+
+<li>express and develop ideas clearly and with increasing accuracy, both orally and in writing</li>
+
+<li>speak coherently and confidently</li>
+
+<li>read literary texts in the language and expand their understanding of the language and culture</li>
+
+<li>write creatively to express their own ideas and opinions</li>
+</ul>
+
+You can explore:
+<ul>
+  <li><a href="https://www.thenational.academy/teachers/programmes/french-secondary-ks3-l/units">examples of French lessons</a></li>
+  <li><a href="https://www.thenational.academy/teachers/programmes/german-secondary-ks3-l/units">examples of German lessons</a></li>
+  <li><a href="https://www.thenational.academy/teachers/programmes/spanish-secondary-ks3-l/units">examples of Spanish lessons</a></li>
+  </ul>
+    </div>
+</section>
+  <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "You build bridges connecting individuals across cultures, fostering understanding, collaboration, and personal growth.",
+    name: "Languages teacher",
+    large: true
+  ) %>
+</section>
+  <section class="col col-space-m col-720">
+     <h2>Check your qualifications</h2>
+<p>To train to teach in primary and secondary schools in England, you’ll need: </p>
+<ul>
+<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary) </li>
+<li>a bachelor’s degree in any subject </li>
+</ul>
+<p>Modern languages teacher training courses have had successful applications from candidates with a range of degrees, including international relations, psychology, socialogy and journalism.</p>
+<p><a href="/non-uk-teachers/train-to-teach-in-england-as-an-international-student">Non-UK citizens can find out more about training to teach in England</a>.</p>
+
+ <p>Your training provider may need you to brush up on your computing knowledge before you become a modern languages teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE)</a> course before you start your teacher training to top up your subject knowledge. </p>
+
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+    <h2> Fund your teaching training </h2>
+
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free scholarships of £27,000 are available for eligible trainee French, German, and Spanish languages teachers</a>. </p>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £25,000 are available for all eligible trainee language teachers, including ancient languages</a>. </p>
+
+  <p>You can receive this alongside a tuition fee and maintenance loan. </p>
```

Comment:
@caitlin-reidy it's talking about bursaries and scholarships so this sounds a bit odd and think it needs to be more specific. You can receive a bursary or scholarship alongside...